sql/index/pilosa/driver.go

```diff
@@ -79,6 +79,15 @@ type (
 func NewDriver(root string) *Driver {
 	h := pilosa.NewHolder()
 	h.Path = filepath.Join(root, "."+DriverID)
+
+	if err := h.Open(); err != nil {
+		panic(err)
+	}
+
+	if err := h.Close(); err != nil {
```

Comment:
why close after open?

Reply:
Because we just need to load into  memory for a new driver and/or loadAll.
We open/close indexes in the lower level (lookup), so if we don't close it then in 
`lookup.Values` we get an error: `invalid view`.

Reply:
Moreover once you Open a holder then the next Open will panic, so we have to Close it